### DISCUSSION TITLE
Forward: Suggestions from others (Feedback)

### DISCUSSION THREAD

**Discussion by pecmens** [mdn/mdn-community#349] [Content]
`box {
        font-size: 14px;
        margin-top: 4px;border-radius: calc(prop(margin-top) * 2); // 8px
}`

It would be great if we could use a method like prop(xxx) to get the value of any property......

This way you don't have to define ugly variables....

**Comment by teoli2003** [comment]
Hello, and thank you for the suggestion!

We are not writing specifications (and don't implement them!). MDN merely documents what browsers are implementing.

If your use case cannot be solved with this [technique](https://stackoverflow.com/questions/66907547/get-and-use-css-property-value-in-css-for-calc#66907857), I suggest you write it down and brings the idea to the **[W3C developer council](https://www.w3.org/developers/get-involved/)**. They have a discourse server, and the council performs initial feedback, helping spec writers to focus on their job.

I hope this will help!

**Comment by bsmth** [comment]
Thanks for starting the discussion. JYP has answered this above so I'm closing for now! Thank you :)
